agent/agent.go

```diff
@@ -114,16 +114,29 @@ func (a *Agent) Backup(bcp pbm.BackupCmd) {
 	}
 
 	log.Printf("Backup %s started on node %s/%s", bcp.Name, nodeInfo.SetName, nodeInfo.Me)
+	tstart := time.Now()
 	err = backup.New(a.pbm, a.node).Run(bcp)
 	if err != nil {
 		log.Println("[ERROR] backup:", err)
 	} else {
 		log.Printf("Backup %s finished", bcp.Name)
 	}
 
-	// wait before release lock in case backup was super fast and
-	// other agents still trying to get lock
-	time.Sleep(1e3 * time.Millisecond)
+	// In the case of fast backup (small db) we have to wait before releasing the lock.
+	// Otherwise, since the primary node waits for `WaitBackupStart*0.9` before trying to acquire the lock
+	// it might happen that the backup will be made twice:
+	//
+	// secondary1 >---------*!lock(fail - acuired by s1)---------------------------
+	// secondary2 >------*lock====backup====*unlock--------------------------------
+	// primary    >--------*wait--------------------*lock====backup====*unlock-----
+	//
+	// Secondaries also may start trying to acquire a lock with quite an interval (e.g. due to network issues)
+	// TODO: we cannot relay on the nodes wall clock.

```

Comment:
```suggestion
	// TODO: we cannot rely on the nodes wall clock.
```
Just a spelling change